### PR TITLE
Add crowdin file

### PR DESCRIPTION
**Pull request by daniserejo**
Add crowdin file for crowdin integration

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```
Comment (camilavcoutinho) [inline CHANGELOG.md]:
```suggestion

### Added
```

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

**Review by camilavcoutinho — APPROVED** [review]

**Review by juliarocha — APPROVED** [review]
🚀

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Crowdin.yml for crowdin integration
+
```
Comment (igorbrasileiro) [inline CHANGELOG.md]:
```suggestion
```

**Review by igorbrasileiro — APPROVED** [review]
Remove changelog changes and merge.

**Pull request closed without merging**